litgpt/data/tinystories.py

```diff
@@ -27,8 +27,9 @@ class TinyStories(DataModule):
     which are the output of the preprocessing step."""
     seed: int = 42
     """The seed to use for shuffling the dataset."""
-    num_workers: int = 8
-    """The number of workers to use for the dataloaders."""
```

Comment:
This was on purpose. Notice that this says "for the dataloaders". The hardcoded value is for the data preparation

During training, too many workers might decrease throughput. Configuring it is useful.

But the data preparation happens before training starts, so I thought it was best to use all CPUs to finish asap.